src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -102,6 +102,10 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
     // DAC is used e.g. for R9M, ES915TX and Voyager
     Radio.SetOutputPower(0b0000);
     TxDAC.setPower((DAC_PWR_)Power);
+
+#elif defined(TARGET_RX_DEFAULT_POWER)
```

Comment:
remove default RX power from end of this list (lines 465...467). This code:
```
#elif defined(TARGET_RX)
     // Set to max power for telemetry on the RX if not specified
    Radio.SetOutputPowerMax();
```

And use something like this:
```
#elif defined(TARGET_RX)
    #if defined(TARGET_RX_DEFAULT_POWER)
        Radio.SetOutputPower(TARGET_RX_DEFAULT_POWER);
    #else
        // Set to max power for telemetry on the RX if default power is not specified
        Radio.SetOutputPowerMax();
    #endif
```

Reply:
Previous definitions work fine.  I tried your suggestion and defined TARGET_RX_DEFAULT_POWER=-10. power management doesn't work.  PA outputs saturation power.

Reply:
Hmm, somwthing wrong with the flaging then. I will take a look...

Reply:
@MATEKSYS maybe you can try this patch. I am pretty sure it will work as expected :)
Copy this snippet into file e.g. `test.patch`
```suggestion
diff --git a/src/lib/POWERMGNT/POWERMGNT.cpp b/src/lib/POWERMGNT/POWERMGNT.cpp
index 416e0beb..c3599bc7 100644
--- a/src/lib/POWERMGNT/POWERMGNT.cpp
+++ b/src/lib/POWERMGNT/POWERMGNT.cpp
@@ -103,9 +103,14 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
     Radio.SetOutputPower(0b0000);
     TxDAC.setPower((DAC_PWR_)Power);
 
-#elif defined(TARGET_RX_DEFAULT_POWER)
-    Radio.SetOutputPower(TARGET_RX_DEFAULT_POWER);
-    
+#elif defined(TARGET_RX)
+    #if defined(TARGET_RX_DEFAULT_POWER)
+        Radio.SetOutputPower(TARGET_RX_DEFAULT_POWER);
+    #else
+        // Set to max power for telemetry on the RX if not specified
+        Radio.SetOutputPowerMax();
+    #endif
+
 #elif defined(TARGET_TX_ESP32_SX1280_V1) || defined(TARGET_RX_ESP8266_SX1280_V1)
     switch (Power)
     {
@@ -456,15 +461,12 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
     case PWR_500mW:
         Radio.SetOutputPower(3);
         break;
-    case PWR_50mW:    
+    case PWR_50mW:
     default:
         Power = PWR_50mW;
         Radio.SetOutputPower(-13);
         break;
     }
-#elif defined(TARGET_RX)
-     // Set to max power for telemetry on the RX if not specified
-    Radio.SetOutputPowerMax();
 #else
 #error "[ERROR] Unknown power management!"
 #endif
```

then you can apply it running `git apply test.patch`, build, flash and check the output...
thanks!

Reply:
@MATEKSYS any update?
We are planning to include this into 1.2 release.

Reply:
> @MATEKSYS any update? We are planning to include this into 1.2 release.

on holiday